src/components/Differential/ScatterPlotDiv.jsx

```diff
@@ -106,12 +121,12 @@ class ScatterPlotDiv extends Component {
 
   handleDropdownChange = (evt, { name, value }) => {
```

Comment:
This function lacked a number of things to properly change state and session storage